server/scheduled/executor_service.go

```diff
@@ -33,11 +33,15 @@ func NewExecutorService(
 	}
 
 	return &ExecutorService{
-		log:                   log,
-		runtimeStatsPublisher: runtimeStatsPublisherJob,
+		log:  log,
+		jobs: []JobDefinition{runtimeStatsPublisherJob},
 	}
 }
 
+func (s *ExecutorService) AddJob(jd JobDefinition) {
```

Comment:
Can we add a test for this? Or do you think it's too small of a function for it to be worth testing?

Reply:
fc9f0fd adds a test for executor service

Reply:
Thank you!